test/models/relation_test.rb

```diff
@@ -230,4 +230,24 @@ def test_changeset_bbox_delete_relation
     assert_equal 39, changeset.min_lat
     assert_equal 116, changeset.max_lat
   end
+
+  # Check that the preconditions fail when you are over the defined limit of
+  # the maximum number of members in a relation.
+  def test_max_members_per_relation_limit
+    # Speed up unit test by using a small relation member limit
+    Settings.max_number_of_relation_members = 20
```

Comment:
This should probably be reset to the original value after the test - it's probably not an issue here but changing global state in a test needs to be done carefully in case it affects later tests.

Reply:
Good point, should be fixed now.